packages/mgt-element/src/Graph.ts

```diff
@@ -155,10 +155,15 @@ export function createFromProvider(provider: IProvider, version?: string, compon
     new HTTPMessageHandler()
   ];
 
-  const client = Client.initWithMiddleware({
-    middleware: chainMiddleware(...middleware)
-  });
+  let baseURL = provider.baseURL ? provider.baseURL : MICROSOFT_GRAPH_DEFAULT_ENDPOINT;
+  const validURL = validateBaseURL(baseURL);
+  if (validURL) {
```

Comment:
We should probably throw an error or at the very least log an error if we got an invalid url here